integration-tests/client-server/src/test/java/io/quarkiverse/cxf/it/redirect/RedirectTest.java

```diff
@@ -70,7 +86,15 @@ void doubleRedirectMaxRetransmits1(String endpoint) {
         int sizeBytes = 16;
         getResponse(endpoint, sizeBytes)
                 .statusCode(500)
-                .body(CoreMatchers.containsString("Unexpected EOF in prolog"));
+                .body(
+                        CoreMatchers.either(
+                                Matchers.matchesPattern("\\QReceived redirection status 307 from"
+                                        + " http://localhost:\\E[0-9]+\\Q/RedirectRest/singleRedirect"
+                                        + " by client doubleRedirectMaxRetransmits1,"
+                                        + " but already performed maximum number 1 of allowed retransmits;"
+                                        + " you may want to increase quarkus.cxf.client.doubleRedirectMaxRetransmits1.max-retransmits."
+                                        + " Visited URIs: http://localhost:\\E[0-9]+\\Q/RedirectRest/doubleRedirect -> http://localhost:\\E[0-9]+\\Q/RedirectRest/singleRedirect\\E"))
+                                .or(CoreMatchers.containsString("Unexpected EOF in prolog")));
```

Comment:
question: why is the second matcher necessary? The test seems to only return the first string. Same for the other tests with two matchers.

Reply:
The same tests are run with `QUARKUS_CXF_DEFAULT_HTTP_CONDUIT_FACTORY=URLConnectionHTTPConduitFactory` environment variable https://github.com/quarkiverse/quarkus-cxf/blob/main/.github/actions/build-and-run-jvm-tests/action.yml#L44-L49
The alternative set of error messages is for the URLConnectionHTTPConduit.

Reply:
Didn't realize that, thank you!